src/App/Components/RecipeCard.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './RecipeCard.css';
+import logo from '../../Assets/logo.png'
+import { Context } from '../../Hooks/Context';
+function RecipeCard({name, image, lm, open, pic, index, heartpresent }) {
+  const { myRecipes, setMyRecipes, setrandchange, randchange } = useContext(Context);
+  const [favColorState, setFavColorState] = useState('#fff')
```

Comment:
This is not the right way to determine this favorite color, what if an item comes marked at favorite already?